extensions/src/org/partiql/extensions/cli/functions/QueryDDB.kt

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at:
+ *
+ *      http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+package org.partiql.extensions.cli.functions
+
+import com.amazon.ion.system.IonReaderBuilder
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder
+import com.amazonaws.services.dynamodbv2.document.ItemUtils
+import com.amazonaws.services.dynamodbv2.model.ExecuteStatementRequest
+import org.partiql.lang.eval.EvaluationSession
+import org.partiql.lang.eval.ExprValue
+import org.partiql.lang.eval.ExprValueFactory
+import org.partiql.lang.eval.stringValue
+import org.partiql.lang.types.FunctionSignature
+import org.partiql.lang.types.StaticType
+
+/**
+ * [QueryDDB] is a function to perform a single query on DDB using PartiQL.
+ * It uses the default configuration provider to establish a connection with AWS. Please reference the official
+ * AWS documentation for specifying which account/profile to use via credentials overrides. Reference the CLI.md
```

Comment:
Perhaps it's worthwhile to move this comment to the default `constructor` as we might end up extending the class to initialize with other cred. providers as well. If going w/ this change a slight change to `README.md` to reflect this would be great as well.

Reply:
Looking at this again, I think we can just remove this from here b/c the creds. are supplied via the client.